app/controllers/insured/inboxes_controller.rb

```diff
@@ -18,6 +19,11 @@ def successful_save_path
     exchanges_hbx_profiles_root_path
   end
 
+  def destroy
```

Comment:
@TristanB17 Is there a reson for adding this endpoint?

Reply:
@saikumar9 it was and endpoint being inherited from InboxesController, but the original endpoint didn't have any MIME type restrictions -> I removed the endpoint and added the MIME restrictions to the base controller after verifying all child controllers were only supposed to respond with JS

My mistake, I thought doing it the way I originally did it was easier to test/a more gradual change.